addons/missionTesting/CfgEden.hpp

```diff
@@ -0,0 +1,197 @@
+                                };
+                                class DASKDAWN{
+                                    name = "DUSK/DAWN";
+                                    value = 2;
+                                };
+                                class MSV{
+                                    name = "MSV";
+                                    value = 3;
+                                };
+                                class TVT1{
+                                    name = "TVT1";
+                                    value = 4;
+                                };
+                                class TVT2{
+                                    name = "TVT2";
+                                    value = 5;
+                                };
+                                class AH{
+                                    name = "AH";
+                                    value = 6;
+                                };
+                            };
+						};
+                        class GVAR(missionTag2) : GVAR(missionTag1){
+                            displayName = "Mission Tag 2:";
+                            property = QGVAR(missionTag2);
+                        };
+                        class GVAR(missionTag3) : GVAR(missionTag1){
+                            displayName = "Mission Tag 3:";
+                            property = QGVAR(missionTag3);
+                        };
+					};
+				};
+				class MissionFlags{
+                    displayName = "Mission Flags";
+                    collapsed = 0;
+					class Attributes{
+						class GVAR(missionFlagCustomScripting){
+                            displayName = "Custom Scripting:";
+                            control = QUOTE(Checkbox);
+							property = QGVAR(missionFlagCustomScripting);
+                            defaultValue = QUOTE(false);
+						};
+						class GVAR(missionFlagCustomLoadout){
+                            displayName = "Custom Loadout:";
+                            control = QUOTE(Checkbox);
+							property = QGVAR(missionFlagCustomLoadout);
+						};
+						class GVAR(missionFlagCustomVicLoadout){
+                            displayName = "Custom Vehicle Loadout:";
+                            control = QUOTE(Checkbox);
+							property = QGVAR(missionFlagCustomVicLoadout);
+						};
+						class GVAR(missionFlagUnitSpecificBriefing){
+                            displayName = "Unit Specific Briefings:";
+                            control = QUOTE(Checkbox);
+							property = QGVAR(missionFlagUnitSpecificBriefing);
+						};
+					};
+				};
+				class MissionMakerNotes{
+                    displayName = "Mission Maker Notes";
+                    collapsed = 0;
+					class Attributes{
+						class GVAR(missionMakerNotesForTesters){
+                            displayName = "Any other notes for Mission Testers:";
+                            control = QUOTE(EditMulti5);
+							property = QGVAR(missionMakerNotesForTesters);
```

Comment:
bad indentation

Reply:
This only shows weird in GitHub. It's fine in VSC..... Don't really know what to do about that.